### DISCUSSION TITLE
Double_point water simulation

### DISCUSSION THREAD

**Discussion by hekungeo** [Anura3D/Anura3D_OpenSource#114] [Q&A] [answered]
Hi Dear all,
I have a problem when processing the water material using the Newtonian constitutive model, like the screenshot below: 
Is there any parameter that should be set in the materials of water? BTW, I am using the double-point type in Anura 2023.
![image](https://github.com/Anura3D/Anura3D_OpenSource/assets/105496180/08ddd5b2-c110-48a8-88d0-b0a12138cbef)
![image](https://github.com/Anura3D/Anura3D_OpenSource/assets/105496180/86efc419-651a-43bd-a935-f18a303e5b35)

Thanks for your time and help.
Cheers
Kun

**Comment by ayerrocolom** [comment, marked as answer]
Hello!
In Anura3D we do not provide a material model called "Newtonian". So, the code does not recognize that.

**Reply by EmreDumlu-UM** [reply]
Dr. Yerro,

I have the same issue. Now I understand why. However, how can we define water material properties if Anura 3D does not recognize that? We are considering to continue our dam-break simulations. Should we continue with the previous version?

**Reply by murphy-john** [reply]
@ayerrocolom and @gdicarluccio I think there is an issue with the GiD script outputting the material model when the GOM file is created. I think when users select "Generate Anura3D files" in GiD, the new problem type is writing 'Newtonian', which isn't recognized by the "ReadMaterialProperties" in the Anura3D code. Instead the GiD script should be changed to write 'newtonian_liquid' which is recognized in by the code.

**Reply by jacobosaa** [reply]
Hi @murphy-john , I've posted some discussions on the forum because I have the very same problem as @Emre-ncche and @hekungeo, but I'm using the Bingham model instead of the Newtonian model. Could you tell me how can I fix that, by re-writing some line on the code, re-arranging the .dll file (A3DBingham.dll) please?
Your time and answer are very appreciated.
Thanks!
Greetings from Chile.

Ps: I took the liberty of posting the image that A3D gives me every time I try to run it. 
![error gom bingham](https://github.com/Anura3D/Anura3D_OpenSource/assets/128545000/2213cda5-8ae5-464a-ac06-4d998b76c4a6)

**Reply by murphy-john** [reply]
In your case, try replacing "Bingham" with "bingham_liquid" in the GOM file where the materials are specified.

**Reply by gdicarluccio** [reply]
Hi all! The Problem Type has been updated to fix the issues with writing the liquid material in the GOM file.

**Comment by EmreDumlu-UM** [comment]
Dear Murphy, Yes. I just figured out this too. I was about send an updated mail. Please find the attached screenshot.  I am appreciated your reply, as well. Best regards,Emre From: John MurphySent: Thursday, November 2, 2023 12:01 PMTo: Anura3D/Anura3D_OpenSourceCc: Emre; CommentSubject: Re: [Anura3D/Anura3D_OpenSource] Double_point water simulation (Discussion ***@***.*** and @gdicarluccio I think there is an issue with the GiD script outputting the material model when the GOM file is created. I think when users select "Generate Anura3D files" in GiD, the new problem type is writing 'Newtonian', which isn't recognized by the "ReadMaterialProperties" in the Anura3D code. Instead the GiD script should be changed to write 'newtonian_liquid' which is recognized in by the code.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>

**Reply by jacobosaa** [reply]
Hi @Emre-ncche My name is Jacobo and i'm modelling a material as a liquid but instead of using the newtonian liquid model, i'm using the Bingham liquid model, and I have very same problem. With the response that @murphy-john gave you, could you fix that problem and proceed to model your dam-break simulation? Please, your answer will be very appreciated. 

Thanks!

**Comment by EmreDumlu-UM** [comment]
Dear Jacobo, I have never used Bingham model however, you should be able to navigate your .A3D files and search for GOM file. Then change material model from “Bingham” to “bingham_liquid”. It should work. Best regards,Emre From: negrojacoboSent: Thursday, November 2, 2023 3:36 PMTo: Anura3D/Anura3D_OpenSourceCc: Emre; MentionSubject: Re: [Anura3D/Anura3D_OpenSource] Double_point water simulation (Discussion #114) Hi @murphy-john , I've posted some discussions on the forum because I have the very same problem as @Emre-ncche and @hekungeo, but I'm using the Bingham model instead of the Newtonian model. Could you tell me how can I fix that, by re-writing some line on the code, re-arranging the .dll file (A3DBingham.dll) please?Your time and answer are very appreciated.Thanks!Greetings from Chile.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>

**Reply by jacobosaa** [reply]
Hello Emre! 
Thank you very much for your time and response, I will immediately check your advice and I'll let you know how it goes.
Again, thank you very much!

Greetings 
Jacobo
